src/ORM/FieldType/DBDate.php

```diff
@@ -34,6 +34,12 @@ class DBDate extends DBField
      */
     const ISO_DATE = 'y-MM-dd';
 
+    /**
+     * Fixed locale to use for ISO date formatting. This is necessary to prevent
+     * locale-specific numeric localisation breaking internal date strings.
+     */
+    const ISO_LOCALE = 'en_NZ';
```

Comment:
Isn't the default i18n locale en_US? Maybe we should use that?

Reply:
NZ is better. Also, we don't want a soft-coded default_locale since that can be changed.